### PR TITLE
Add opacity to home page positions when loading

### PR DESCRIPTION
**Pull request by mjoyce91**
Addresses #580 by adding opacity to the containers for New and Popular positions on the home page when they're loading.

### CODE REVIEW

**Comment by burgwyn** [comment]
This seems like the perfect item to document with gifs

**Comment by mjoyce91** [comment]
b7r4vktpyv

File: src/sass/_spinner.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@ $tm-spinner-filled-ratio: 1.15;
 
 .tm-spinner {
   color: $tm-navy;
+  z-index: 100; // make sure spinner is on top
```
Comment (burgwyn) [inline src/sass/_spinner.scss]: Let's extract the `z-index` into a global variable so we can better track all the potential `z-index` values we have/may have.

**Comment by mjoyce91** [comment]
@burgwyn I can see that as a possibility - multiple lists loading on the same page.

**Pull request merged**